src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.VisualBasic.targets

```diff
@@ -17,15 +17,18 @@ Copyright (c) .NET Foundation. All rights reserved.
   <PropertyGroup>
     <DefineConstants>$(DefineConstants),$(ImplicitFrameworkDefine)</DefineConstants>
   </PropertyGroup>
-  <ItemGroup>
+  <ItemGroup Condition=" '$(DisableImplicitFrameworkReferences)' != 'true' and '$(TargetFrameworkIdentifier)' == '.NETFramework'">
+    <!-- These namespaces are present in 2.0 Framework assemblies -->
     <Import Include="Microsoft.VisualBasic" />
     <Import Include="System" />
     <Import Include="System.Collections" />
     <Import Include="System.Collections.Generic" />
     <Import Include="System.Data" />
```

Comment:
I'd say just drop System.Data altogether since that's not in the default set of referneces for netstandard\netcore